### ISSUE TITLE
"cabal fetch" should take output arguments

### ISSUE THREAD

**Issue by bos** [haskell/cabal#446]
(Imported from [Trac #453](http://hackage.haskell.org/trac/hackage/ticket/453), reported by @gregorycollins on 2009-01-12)

It would be nice if "cabal fetch" could output the fetched tarball to a location specified on the command line, i.e.:

<blockquote>
cabal fetch bytestring -o foo.tgz

</blockquote>

This probably depends on #423.

**Comment by jsl** [comment]
Given that the reporter doesn't need this any more, and there is no other activity since 2009, I propose closing. Please re-open or create a new ticket if this is still desired.

/cc @tibbe

**Comment by gregorycollins** [comment]
We should put a TODO in the unpack code to add this, however, even if the ticket is closed.

**Comment by jsl** [comment]
Thanks for following up on this @gregorycollins. I just flagged this since it seemed to have stagnated, and I'm trying to help clean up the backlog a bit. After looking at it more closely, this definitely does seem like something cabal should have. Let's leave this issue open - I'll take a stab at implementing this when I get a chance.

For reference I checked other popular package managers, Ruby gems allows specifying output directory with `--target`, npm extract allows `-o` and `--out`, and pip install supports `-t` and `--target`.

**Comment by gregorycollins** [comment]
Yes, it was an obvious oversight not to include this from the beginning, however it's not so important and nobody has had the time.
